cmd/start.go

```diff
@@ -29,10 +29,20 @@ func init() {
 }
 
 func runStart(cmd *cobra.Command, args []string) {
+	checkDaemon()
 	handleGlobalNetwork()
 	startContainers(startFlags.Services)
 }
 
+func checkDaemon() {
```

Comment:
Instead of `checkDaemon`, maybe we would benefit on renaming this function to something like `checkKoolDependencies`.